src/types.ts

```diff
@@ -0,0 +1 @@
+export type Numeric = number|string;
```

Comment:
i don't think you should have this in a generic location this unless it explicitly fills the contract in a generic yet explicit way...can you remove this and move this into the file where it's being used (or just use `number | string` directly without defining a custom type)?

Reply:
Done.